test/scripts/setup-konvoy.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+KONVOY_VERSION="v1.4.0"
```

Comment:
Shouldn't we test v1.4.1 by now?

Reply:
That one doesn't appear to be published yet, but we can change this in CI without having to change the code when we need to.

Reply:
https://wiki.d2iq.com/display/ENG/Download+link
What do you mean its not published, those links would be the S3 link you would use right?

Reply:
updated!